src/screens/LinkContextScreen/elements/Header.js

```diff
@@ -186,90 +171,90 @@ const styles = StyleSheet.create({
     paddingRight: 20,
     alignItems: 'center',
     borderBottomWidth: 0.5,
-    borderBottomColor: COLORS.gray1,
+    borderBottomColor: COLORS.balance_gray,
     paddingBottom: 8,
     paddingTop: 8,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.white
   },
   wrapperImage: (showBackButton = true) => ({
     borderRadius: 45,
     borderWidth: 0.2,
-    borderColor: 'rgba(0,0,0,0.5)',
+    borderColor: COLORS.black50,
     width: 48,
     height: 48,
     justifyContent: 'center',
     alignItems: 'center',
-    marginLeft: showBackButton ? 0 : 20,
+    marginLeft: showBackButton ? 0 : 20
   }),
   image: {
     height: 48,
     width: 48,
-    borderRadius: 45,
+    borderRadius: 45
   },
   headerDomainName: {
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[600],
     lineHeight: normalizeFontSize(16.9),
-    color: '#000000',
+    color: COLORS.black
   },
   headerDomainDate: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18),
-    color: '#828282',
-    flexShrink: 1,
+    color: COLORS.blackgrey,
+    flexShrink: 1
   },
   headerFollowerText: {
-    color: '#828282',
+    color: COLORS.blackgrey,
     fontSize: 12,
-    fontFamily: fonts.inter[700],
+    fontFamily: fonts.inter[700]
   },
   domainItemTitle: {
     fontSize: 16,
     fontFamily: fonts.inter[700],
-    lineHeight: 24,
+    lineHeight: 24
   },
   point: {
     width: 3,
     height: 3,
     borderRadius: 4,
-    backgroundColor: COLORS.gray,
+    backgroundColor: COLORS.blackgrey,
     marginLeft: 8,
-    marginRight: 8,
+    marginRight: 8
   },
   domainIndicatorContainer: {
-    marginLeft: -4,
+    marginLeft: -4
   },
   wrapperText: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.white,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.anon_primary,
     width: 28,
     height: 28,
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: 0.5,
+    borderWidth: 0.5
   },
   wrapperTextUnFollow: {
-    backgroundColor: '#00ADB5',
+    backgroundColor: COLORS.anon_primary,
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.anon_primary,
     width: 28,
     height: 28,
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: 0.5,
+    borderWidth: 0.5
   },
   leftRowContainer: {
     flex: 1,
     flexDirection: 'row',
-    display: 'flex',
+    display: 'flex'
   },
   backbutton: {
     padding: 16,
     paddingHorizontal: 20,
-    alignSelf: 'center',
-  },
+    alignSelf: 'center'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In this hunk, several color values have been replaced with references to the new `COLORS` object. This is part of the color theme refactoring and should improve consistency across the application. However, similar to the previous comment, please ensure that all new color references match their old values to maintain the same visual appearance.


<!-- This is an auto-generated comment by OSS CodeRabbit -->